src/gsy_e/gsy_e_core/export.py

```diff
@@ -66,6 +66,26 @@
     3: "net_metering_pricing"
 }
 
+json_data_area_fields_mapping = {
```

Comment:
Minor, but I propose to rename this into:
`json_results_file_name_mapping`

Reply:
Actually, I used the exact name as gsy-web, so people can find about this one if they try searching for `json_data_area_fields_mapping` in GitHub (This is what I always do when I want to know which repo is using which other repo).
As, I didn't write `json_data_area_fields_mapping` in gsy-web, I preferred to not change it there, too!
But, still, if you think the name is misleading people, I can change it in gsy-e, maybe?

Reply:
Sorry, I did not see it in the gsy-web PR.
Actually, in gsy-web it is missleading, too. There I would rename it to e.g. `results_field_to_json_filename_mapping`. 
I do not have a strong opinion on exactly these names, but don't you agree that these (or a version of them) would be more explanatory? I agree, they should be the same in both repos. So please change them on both places, thanks a lot!

Reply:
You're right. both names are misleading. I will change the name and ask for another round of review.